htmd/parameterization/fftype.py

```diff
@@ -3,31 +3,58 @@
 # Distributed under HTMD Software License Agreement
 # No redistribution in whole or part
 #
-
-import shutil
-import subprocess
 import os
+import logging
```

Comment:
Please do _not_ do this. There's a reason I did lazy import, it's for the cli not to _slow_.
But OK, I guess it's my fault for not adding a speed test. I've just created one and I am pushing it to this branch. Then, I leave it to you to pass the test.

Reply:
```
$ time python -c 'import htmd.parameterization.fftype'

Copyright by Acellera Ltd. By executing you are accepting the License. In order to register, run htmd_register on your terminal


HTMD Config file /shared/raimis/opt/acellera/conf.py executed.

real	0m0.607s
user	0m0.268s
sys	0m0.103s
```